app/scripts/lib/country-telephone-info.js

```diff
@@ -70,7 +70,7 @@
          }
          return `+1${num}`;
        },
-       pattern: /^(?:\+?1)?\d{10,10}$/, // allow for a +1 or 1 prefix before the area code
+       pattern: /^(\+?1)?[2-9]\d{9,9}$/, // allow for a +1 or 1 prefix before the area code, area codes are all 2-9
```

Comment:
@philbooth - my medium term goal is to create one repo with regexp patterns that can be shared across repos. The # of different validation rules is a foot gun waiting to happen.